teacher.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(specialization, age, name, parent_permission)
```

Comment:
- [ ] the `name` and `parent_permission` arguments should have a default value similar to in the `Person` class